process/transaction/shardProcess.go

```diff
@@ -605,6 +589,7 @@ func (txProc *txProcessor) removeValueAndConsumedFeeFromUser(
 	if err != nil {
 		return err
 	}
+	//TODO: check if always the consumedFee was gasLimit*GasPrice - movebalance for example
```

Comment:
yes it is. this is somewhat changed in gas price modifier.

Reply:
ok